react/utils/blocks.ts

```diff
@@ -0,0 +1,79 @@
+export const getBlockRole = (treePathTail: string) => {
```

Comment:
You could have created a type for `'after' | 'around' | 'before'` and use that as the return type here, probably something like `(treePathTail: string): BlocksEnforcedByInterface | null` (Not sure about the name, but well...).

WDYT?

Reply:
I agree that creating a type for block roles would make the return type more semantic — and I'll do that —, but I still need to use `as` in the return statement, because `match` returns a generic `string`.